README.md

```diff
@@ -40,7 +40,7 @@ it is unchecked. At this point your IDE will work with its regular keyboard
 shortcuts.
 
 Keyboard shortcut conflicts between the Vim emulation and the IDE can be
-resolved via "File | Settings | Vim Emulation", "File | Settings | Keymap"
+resolved via "Preferences | Other Settings | Vim Emulation", "Preferences | Keymap"
```

Comment:
@jamescmartinez Thanks for your pull request. Actually it's File | Settings | ... on Linux and Windows and Preferences | ... on macOS. Could you please mention macOS specific names separately?